### DISCUSSION TITLE
Using VMware workstation 16 pro do determine what the management IP and gateway should be used

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5652] [Ideas] [answered]
I am using windows 10 on my laptop for the host. And the latest SO iso "WAHZU 2.3.70". 
  I have been trying to get a successful install since June. I am new to cyber security and to security onion. My first install seemed to work, but i think i have tried to do about 30 or so installs since June, yes, while waiting the 30+ minutes for everything to install only to get install issues that reffered me to the sosetup and errors log files. my errors.log was always empty and i had no clue where to start looking in the sosetup.log. 
  But during the last two days with the help and advice from you guys i successfully installed an import, standalone and eval VM. 
  YEA!!
   So I thought i would describe my experience and explain what i finally did. 
   It seems like the complete install needs good IP addresses to get information it needs from github or other places during the install. 
   for this  
![image](https://user-images.githubusercontent.com/5569263/134785755-cb64288a-bce0-4c9d-8087-44be6060cb50.png)
I went to my command prompt in windows 10 and got the NAT VMnet8 IPv4
![image](https://user-images.githubusercontent.com/5569263/134785825-38e50843-599b-41fa-b8c5-ac3de7600523.png)

for this
![image](https://user-images.githubusercontent.com/5569263/134785764-dc4e395c-be38-479b-8c4c-ef5d725346b5.png)
 I went into the VMware workstation 16 pro>edit>Virtual Network Editor
![image](https://user-images.githubusercontent.com/5569263/134785906-e77f5312-2e82-4b01-b40d-19c13a616f9f.png)

and for this, i used an IP address from a VMnet that was not used elswhere, in my case i used VMnet5 (.50.)
![image](https://user-images.githubusercontent.com/5569263/134785784-9628c460-dbd4-4c20-aa32-9606315622d7.png)
  But now I am having issues with getting the SOC to come up, but thats my new challange. 
  I am  just hoping this can help someone. 
  In the mean, thanks for your help with this one.
  I am putting this in the ideas categories, not sure where to put it for now.
  In fact, I wish i knew this while installing from " https://docs.securityonion.net/en/2.3/first-time-users.html " for those using windows 10 host and VMware workstation 16 pro.
  thanks again !!

**Comment by n0th1nt0lOOz** [comment]
I think you need to learn network basic first before get into an NSM

**Comment by iqworks** [comment]
HA, how much of the basics of networking do you suggest I know to start this post? That’s right, you can’t even tell me that. I will hold my true thoughts here about your reply 😊

This post is just to try to help others using the tools I have used in order to determine the IP addresses I used to get a complete install of security onion. 

I was not asking for information or a tutorial from an export. I was simply suggesting something I actually did that actually helped me to achieve a successful SO install so others might 

Have some help.

**Comment by dougburks** [comment, marked as answer]
If Windows is already using 192.168.80.1, then you probably don't want to assign that same IP address to your VM.

**Reply by iqworks** [reply]
I guess the fact that i see 192.168.80.1 in my ipconfig means that windows is already using it? My next choice would be 192.168.80.0/24, could i use this as my management IP? 
  I entered my ubuntu IP address in so-allow. 
right now when i try to use 192.168.80.1 from an ubuntu browser i get a blank screen, perhaps this is because windows is using it? Then again there could be any number of reasons for that.
  I cannot seem to find info about what the management IP does inside security onion so I can get a better handle on what is needed there. Can you point me to a link(s)?
thanks for your help

**Reply by dougburks** [reply]
> I guess the fact that i see 192.168.80.1 in my ipconfig means that windows is already using it?

Yes.

> My next choice would be 192.168.80.0/24, could i use this as my management IP?

If the subnet mask is /24, then the .0 address is usually considered to be the network address and would not be assigned to an individual device in the network.

> right now when i try to use 192.168.80.1 from an ubuntu browser i get a blank screen, perhaps this is because windows is using it?

Yes, you're trying to connect to a web server on your Windows host but there is no web server there.

> I cannot seem to find info about what the management IP does inside security onion so I can get a better handle on what is needed there. Can you point me to a link(s)?

Just like your Windows host and your Ubuntu VM, Security Onion needs an IP address to communicate with other hosts. For more information, please see:
https://docs.securityonion.net/en/2.3/hardware.html#nic

Finally, we've discussed previously that basic networking knowledge is a prerequisite for cybersecurity in general and that basic networking questions are beyond the scope of this discussions forum:
https://github.com/Security-Onion-Solutions/securityonion/discussions/5131
https://github.com/Security-Onion-Solutions/securityonion/discussions/5134

Unfortunately, we cannot continue to teach you networking basics in this forum. Please consider studying networking on your own time.

Thanks.

**Comment by iqworks** [comment]
Ok, I will, thanks for your help thus far.

**Comment by iqworks** [comment]
I do not mean to drag you guys into answering questions about basic networking issues. Its just that for me right now, security onion and basic networking are a blur to me right now because all of this is new to me. 

But, I will differentiate where they both begin and end. Security onion allows you to use tools for various cybersecurity monitoring etc. I will continue to learn about those networking basics so I do not cross the lines of this forum. 

After all, its only a computer and only software 😊
